Spec/RealtimeClientConnection.swift

```diff
@@ -3560,16 +3560,19 @@ class RealtimeClientConnection: QuickSpec {
                     TestProxyTransport.network = .hostUnreachable
                     defer { TestProxyTransport.network = nil }
 
-                    var urlConnections = [NSURL]()
+                    var urls = [NSURL]()
```

Comment:
We can just use `URL` instead of `NSURL` and avoid those casts.

Reply:
Fixed at b13ddb51b7c159e6f784ee0f837f1d9910377922.